Src/main.c

```diff
@@ -379,13 +379,10 @@ float consumed_current = 0;
 int32_t smoothed_raw_current = 0;
 int16_t actual_current = 0;
 
-char lowkv = 0;
```

Comment:
the PR includes changes to other code, I presume by accident. It is a good idea to review the changes files in your PRs :-)